### PR TITLE
enhancements: Update monitoring alerting consistency proposal

### PR DESCRIPTION
**Pull request by bison**
This enhancement proposal was initially added in #637. There was lots of discussion at the time, and it was decided at some point that it would be merged as-is to provide a starting point.

While the original proposal absolutely provided a great jumping-off point to the discussion around issues with alerting in OpenShift, the monitoring team would like to see it now evolve into a more implementable enhancement proposal focused on developing a practical style guide for OpenShift alerts. We'd also like to see the acceptance criteria for new critical alerts in OpenShift formalized here. This is a first pass at reshaping the original proposal into something in this direction.

### CODE REVIEW

**Comment by bison** [comment]
cc: @openshift/sre-alert-sme @openshift/openshift-team-monitoring

File: enhancements/monitoring/alerting-consistency.md [hunk 1]
```diff
@@ -99,37 +140,39 @@ alerts so they fire before the situation becomes irrecoverable. Expect users to
...
+This alert fires if no Kubernetes API server instance has reported metrics
+successfully in the last 15 minutes.  This is a clear example of a critical
+control-plane issue that represents a threat to the operability of the cluster
+as a whole, and likely warrants paging someone.  The alert has clear summary and
+description annotations, and it links to a runbook with information on
+investigating and resolving the issue.
+
+The group of critical alerts should be small, very well defined, highly
+documented, polished and with a high bar set for entry.  This includes a
 mandatory review of a proposed critical alert by the Red Hat SRE team.
```
Comment (bison) [inline enhancements/monitoring/alerting-consistency.md:167]: We should add details on how and where this happens? How does someone contact the SRE team for this review?

**Comment by simonpasquier** [comment]
/lgtm
/cc @openshift/sre-alert-sme

Thanks for doing this!

**Comment by bison** [comment]
Ping @michaelgugino and @sdodson -- Happy to get more feedback here, but not sure how long to wait.

**Comment by dgrisonnet** [comment]
Great effort, thank you for looking into this @bison.

/lgtm

**Comment by sdodson** [comment]
@bison we've waited long enough, thanks
/hold cancel

**Comment by bison** [comment]
Thanks everyone. Of course, if there ends up being more feedback, please reach out to me. We can keep iterating on this.

**Pull request merged**